lib/service_actor/checks/must_check.rb

```diff
@@ -53,11 +53,17 @@ def initialize(input_key:, actor:, nested_checks:, value:)
     @value = value
   end
 
-  def check
+  def check # rubocop:disable Metrics/MethodLength
     @nested_checks.each do |nested_check_name, nested_check_conditions|
       check, message = define_check_and_message_from(nested_check_conditions)
 
-      next if check.call(@value)
+      begin
+        next if check.call(@value)
+      rescue StandardError => e
+        message =
+          "The \"#{@input_key}\" input on \"#{@actor}\" has an error " \
+            "in the code inside \"#{nested_check_name}\": #{e}"
```

Comment:
It might be good to have the error class here, as well.

Reply:
Done.